tests/zapPage.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, test, type Page } from '@playwright/test';
+import { qase } from 'playwright-qase-reporter';
+import {
+    clickItemInSettingsMenu,
+  } from './testData/settingsFunctions'
+import { SETTINGS_MENU } from './testData/testConstants';
+import values from './testData/values.json' assert { type: 'json' };
+import { checkTabsInHeader } from './testData/menuFunctions';
+
+ async function verifyBridgesCount(page: Page, numberOfBridges: number): Promise<void> {
+    const widgetContainer = page.getByTestId('zap-widget-container');
+    const listOfBridges = widgetContainer.locator('ul.MuiList-root.MuiList-padding div[role="button"]');
+    await expect(listOfBridges).toHaveCount(numberOfBridges);
+  }
+
+test.describe('Zap Morpho Katana Page', () => {
+    test.beforeEach(async ({ page }) => {
+        await page.goto('/zap/morpho-katana-usdc');
```

Comment:
Are we sure we'll use `/zap/morpho-katana-usdc` and not `/missions/morpho-katana-usdc`